### ISSUE TITLE
Unable to Reproduce Training results of CDD11

### ISSUE THREAD

**Issue by cheng221** [eduardzamfir/MoCE-IR#5]
![Image](https://github.com/user-attachments/assets/5eb84aa9-ab46-4eec-af8a-5f6886f09ea3)

![Image](https://github.com/user-attachments/assets/327eda36-db26-479d-bde8-5d3a772b03a6)
Thank you for sharing your work and making the code available. I have been trying to reproduce the training results of CDD11 as reported in your paper, but I am encountering some discrepancies. I train MoCE-IR-S with 200 epochs on cdd11-all and test it，but achieve much more lower results on cdd11-haze-snow, could you explain it futher or give me a correct training details to reproduce it, thanks!

**Comment by eduardzamfir** [comment]
hi, 
thanks for your interest. This is unfortunate, do you achieve similar results on other degradations? I double-checked my checkpoint and the numbers in the paper, you can also find the trained checkpoint [here](https://drive.google.com/drive/folders/1pQBceb8cCPdIzbqbNNGqV5qNXzzqL4uK?usp=share_link). 

The training command I posted is missing the correct FFT loss weight set to `--fft_loss_weight 0.1`, as the default weight is set to 1.0, maybe train again with explicitly setting the weight to 0.1

**Comment by wasdone** [comment]
> > ![Image](https://github.com/user-attachments/assets/5eb84aa9-ab46-4eec-af8a-5f6886f09ea3)
> > ![Image](https://github.com/user-attachments/assets/327eda36-db26-479d-bde8-5d3a772b03a6) Thank you for sharing your work and making the code available. I have been trying to reproduce the training results of CDD11 as reported in your paper, but I am encountering some discrepancies. I train MoCE-IR-S with 200 epochs on cdd11-all and test it，but achieve much more lower results on cdd11-haze-snow, could you explain it futher or give me a correct training details to reproduce it, thanks!
> 
> Hello, by setting --fft_loss_weight 0.1, did you reproduce successfully?

@

> ![Image](https://github.com/user-attachments/assets/5eb84aa9-ab46-4eec-af8a-5f6886f09ea3)
> 
> ![Image](https://github.com/user-attachments/assets/327eda36-db26-479d-bde8-5d3a772b03a6) Thank you for sharing your work and making the code available. I have been trying to reproduce the training results of CDD11 as reported in your paper, but I am encountering some discrepancies. I train MoCE-IR-S with 200 epochs on cdd11-all and test it，but achieve much more lower results on cdd11-haze-snow, could you explain it futher or give me a correct training details to reproduce it, thanks!

@cheng221 
Hello, by setting --fft_loss_weight 0.1, did you reproduce successfully?

**Comment by aodwals** [comment]
How are the training set and test set of cdd11 divided? Thanks.
Why is it randomly selected in the dataset?
